bootstrap/Dockerfile

```diff
@@ -46,29 +50,13 @@ RUN cd /opt/ksonnet && \
     cp /tmp/go/bin/ks /usr/local/bin && \
     chmod a+rx /usr/local/bin/ks
 
-# Pull kubeflow ksonnet repos
-
-# Ksonnet needs to have kubernetes env setup even for non-k8s actions.
-# TODO (inc0): remove this when we move to disk registry
-COPY kubeconfig_fake.yml /root/.kube/config
-
-ARG github_token
-
-# TODO (inc0): remove this when we move to disk registry
-RUN export GITHUB_TOKEN=${github_token} && cd / && ks init kubeflow && \
-    cd kubeflow && \
-    ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow && \
-    ks pkg install kubeflow/core && \
-    ks pkg install kubeflow/tf-serving && \
-    ks pkg install kubeflow/tf-job && unset GITHUB_TOKEN
-COPY deploy.sh /kubeflow
-
-RUN mkdir -p /opt/kubeflow
-
 COPY --from=builder /opt/kubeflow/bootstrapper /opt/kubeflow/
 COPY start.sh /opt/kubeflow/
 
-RUN chmod a+rx /opt/kubeflow/*
+RUN chmod a+rx /opt/kubeflow/bootstrapper
+RUN chmod a+rx /opt/kubeflow/start.sh
+
+RUN apt-get update && apt-get install -y vim
```

Comment:
why do we need vim? To edit app?

Reply:
yeah user might want to view or edit their apps?

Reply:
I guess we can keep it here, not a big deal.